mlflow/store/tracking/file_store.py

```diff
@@ -348,17 +355,14 @@ def create_experiment(self, name, artifact_location=None, tags=None):
         self._check_root_dir()
         _validate_experiment_name(name)
         self._validate_experiment_does_not_exist(name)
-        # Get all existing experiments and find the one with largest numerical ID.
-        # len(list_all(..)) would not work when experiments are deleted.
-        experiments_ids = [
-            int(e)
-            for e in (
-                self._get_active_experiments(full_path=False)
-                + self._get_deleted_experiments(full_path=False)
+        experiment_id = _generate_unique_integer_id()
+        if not self._has_experiment(FileStore.DEFAULT_EXPERIMENT_ID):
+            warnings.warn(
+                "The default experiment is not present due to a file system issue or manual "
+                "deletion. Recreating the default experiment with "
+                f"`experiment_id={FileStore.DEFAULT_EXPERIMENT_ID}`."
```

Comment:
Why do we need to create the default experiment in `create_experiment`? Is there an issue if we create a new experiment without the default experiment?

Reply:
I removed this warning and the call to recreate.